_issues/hillary_lgbt.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "Doesn't she just pay lip service to the LGBT community?"
```

Comment:
We've been using lowercase titles so that we add "...". I change these to properly capitalized when displaying the title for social media and in the titles. Could you make it lowercase here?